hangman/test/hangman_test.exs

```diff
@@ -0,0 +1,128 @@
+defmodule HangmanTest do
+  use ExUnit.Case
```

Comment:
## +8

Nice tests.

I really appreciate the decision you made to make `init_state` public so you could simplify the testing. Good choice.

Reply:
Thank you so much, I was even concerning you would minus my points for this. At the beginning of my coding, it was made as a private function on purpose. However, it would be too hard for me to test, with all random words.